frontend/src/components/GMainNavigation.vue

```diff
@@ -497,12 +504,36 @@ function highlightProjectWithKeys (keyDirection) {
 }
 
 function scrollHighlightedProjectIntoView () {
-  if (refProjectListItems.value) {
+  scrollProjectIntoView(highlightedProjectName.value)
+}
+
+function scrollSelectedProjectIntoView () {
```

Comment:
I think we should delete these two functions `scrollHighlightedProjectIntoView` and `scrollSelectedProjectIntoView` and inline the code. From my point of view readability is better for `  scrollProjectIntoView(highlightedProjectName.value)`. What do you think?

Reply:
✅